parking_lot/parking_lot.py

```diff
@@ -1,8 +1,150 @@
 # -*- coding: utf-8 -*-
-from flask import Flask
+import datetime
 
-app = Flask(__name__)
 
-@app.route("/")
-def hello_world():
-    return "<p>Hello, World!</p>"
+class Parking_lot_error( Exception ):
+    message = 'unhandle error with parking lot'
+
+    def __init__( self, message=None ):
+        if message is None:
+            message = self.message
+        self.data = {
+            'status': 'error',
+            'error': message
+        }
+
+
+class Full_parking_error( Parking_lot_error ):
+    message = 'No free space.'
+
+
+class Cannot_find_car_error( Parking_lot_error ):
+    message = 'Cannot find a car in the location {}.'
+
+    def __init__( self, location=None ):
+        message = self.message.format( location )
+        self.data = {
+            'status': 'error',
+            'error': message
+        }
+
+
+class Car:
+    def __init__(
+            self, car, tariff, location, hourly_fee, daily_fee,
+            start=None, finish=None ):
+        self.car = car
```

Comment:
a car should have a license_plate instead of a car.

Reply:
i use **car** because was the key in the rest api, i going to do the change because make more sense.